app/services/overall_metrics_service.rb

```diff
@@ -0,0 +1,13 @@
+module OverallMetricsService
```

Comment:
```suggestion
module GlobalMetricsService
```

I wonder if this is a clearer name for this service.